db/view/view.cc

```diff
@@ -1127,6 +1169,9 @@ future<stop_iteration> view_update_builder::on_results() {
                 cr.apply(std::max(_existing_partition_tombstone, _existing_current_tombstone));
             });
             generate_update(std::move(*_update).as_clustering_row(), { std::move(*_existing).as_clustering_row() });
+        } else if (_update->is_static_row()) {
+            assert(_existing->is_static_row());
```

Comment:
let's go with on_internal_error here and avoid crashing the server

Reply:
Done